proto/p4/v1/p4runtime.proto

```diff
@@ -741,3 +744,13 @@ enum SdnPort {
   // 0xfffffffd: Send to CPU
   SDN_PORT_CPU = -3;
 }
+
+//------------------------------------------------------------------------------
+message CapabilitiesRequest {
+}
+
+message CapabilitiesResponse {
+  // The full semantic version string (e.g. "1.1.0-rc.1") corresponding to the
+  // version of the P4Runtime specification currently implemented by the server.
+  string p4runtime_version = 1;
```

Comment:
This sounds like it's the version of the p4runtime server (which presumably also has a version).  Would p4runtime_language_version or p4runtime_protocol_version be better?

Reply:
How about `p4runtime_api_version` since it is shorter and matches the terminology used in this webpage that we reference: https://cloud.google.com/apis/design/versioning

Reply:
That's an even better suggestion than mine, I like it. :)

Reply:
Done.